### DISCUSSION TITLE
What is Omny-Key?

### DISCUSSION THREAD

**Discussion by HSniBBy** [Lailloken/Exile-UI#318] [Q&A]
Hey guys,

I'm new here and I'm trying to use the tool but it's not working as it should.

There is always talk about (Omni-Key)
Can someone explain to me what the omni-key is supposed to be?
Is it something like a button I have to press to make something happen? If so how or where do you change it to another key if needed. I would like to play Betrayal but I can't get it to work with the tool.

**Comment by Lailloken** [comment]
It's explained in the [general usage guide](https://github.com/Lailloken/Lailloken-UI/wiki#general-usage) that you probably skipped
<br>

![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/a642204f-9f4a-45db-9901-7ba2f40fc491)
